bin/reset-argocd-password

```diff
@@ -1,17 +1,29 @@
 #!/bin/bash
 
+function patch_password() {
+	ENCODED_HASH="$1"
+
+	kubectl -n argocd patch secret argocd-secret \
+	  --type='json' \
+	  --patch="[{'op': 'replace', 'path': '/data/admin.password', 'value': '${ENCODED_HASH}'}]"
+}
+
 PASSWORD=$(pwgen -N1 -Bs 32)
```

Comment:
Unrelated to this change, but is pwgen a commonly installed tool? I've never heard of it yet, and it's not installed on my system. Maybe we can get away with reading from `urandom` which should work kind of everywhere

```
tr -dc A-Za-z0-9 </dev/urandom | head -c 13
```

Feel free to disregard this.

Reply:
I think it's [fairly common](https://manpages.ubuntu.com/manpages/noble/man1/pwgen.1.html) but I still like your suggestion, thanks :sparkler: